ClubManagement/ClubManagement/Controllers/FirebaseController.cs

```
@@ -19,32 +21,50 @@ protected FirebaseController()
 
         public async void Add(T t)
         {
-            await FirebaseClient.PostAsync(t);
+            try
+            {
+                await FirebaseClient.PostAsync(t);
+            }
+            catch { }
```

Please don't ignore the error